automation/deploy-bin.ts

```diff
@@ -64,7 +62,7 @@ export async function release() {
 
 /** Return a cached Octokit instance, creating a new one as needed. */
 const getOctokit = _.once(function() {
-	const Octokit = (require('@octokit/rest') as typeof _ocktokit).plugin(
+	const Octokit = (require('@octokit/rest') as typeof import('@octokit/rest')).Octokit.plugin(
```

Comment:
This change was required due to types, but either form would work fine if I cast to `any`:
```
require('@octokit/rest')
[Function: DeprecatedOctokit] {
  Octokit: [Function: bound Octokit] { plugin: [Function: bound registerPlugin] },
  plugin: [Function: bound registerPlugin]
}
```

Reply:
I'm assuming in 17.x they might remove the `.plugin` form in favor of `.Octokit.plugin` but I haven't checked